tools/build-kani/src/main.rs

```diff
@@ -19,10 +19,7 @@ fn main() -> Result<()> {
     let args = parser::ArgParser::parse();
 
     match args.subcommand {
-        parser::Commands::BuildDev(build_parser) => {
-            build_lib();
-            build_bin(&build_parser.args);
-        }
+        parser::Commands::BuildDev(build_parser) => build_lib().and(build_bin(&build_parser.args)),
```

Comment:
I would've used the [question mark operator](https://doc.rust-lang.org/rust-by-example/std/result/question_mark.html):
```rust
build_lib()?;
build_bin(&build_parser.args);
```
but your version looks cleaner.

Reply:
If you don't want to modify `build_bin`, you can use `and_then` instead of `and`:
```rust
build_lib().and_then(|_| build_bin(&build_parser.args))
```

Reply:
I actually tried the above at first, but this wouldn't compile.